src/await_cps/ioc.clj

```diff
@@ -39,20 +39,30 @@
            ~(invert (assoc ctx :r cont) asn))))
       (then coll))))
 
+(defn add-env-syms [ctx syms]
+  ;; This adds mappings to "true" into the environment map. This doesn't quite
+  ;; match what the Clojure compiler does, but I think it's already recommended
+  ;; that macros don't depend on the values in the &env map.
+  (update ctx :env (fnil into {}) (map (fn [sym] [sym true])) syms))
```

Comment:
What does `(resolve env sym)` ([line 4](https://github.com/mszajna/await-cps/pull/1/files#diff-4944141e3afac9160873219948a5bebcR4)) do with symbols mapped to `true`?

Reply:
It appears to be resolving to `nil` but I can see now it doesn't matter as you only care for it not to resolve to certain vars.